ql/src/java/org/apache/hadoop/hive/ql/ppd/SyntheticJoinPredicate.java

```diff
@@ -94,8 +95,8 @@ public ParseContext transform(ParseContext pctx) throws SemanticException {
     // rule and passes the context along
     SyntheticContext context = new SyntheticContext(pctx);
     SemanticDispatcher disp = new DefaultRuleDispatcher(null, opRules, context);
-    SemanticGraphWalker ogw = new PreOrderOnceWalker(disp);
-
+    PreOrderOnceWalker ogw = new PreOrderOnceWalker(disp);
+    ogw.excludeNode(LateralViewForwardOperator.class);
```

Comment:
Pls add a comment about why the LVF operator is treated in a special way in the context of SyntheticJoinPredicate.  The description in the Jira is nice but would be useful to have a summarized couple of lines here.